app/Http/Controllers/User/HomeController.php

```diff
@@ -14,27 +14,27 @@ class HomeController extends Controller
 {
     public function index(Request $request)
     {
-    	$request = $request->all();
-    	$teachers = Teacher::orderBy('id','ASC');
-    	if (isset($request['grades'])) {
-    		$teacherIds = [];
-    		foreach ($teachers as $key => $teacher) {
-    			$grade = array_intersect($request['grades'], $teacher->grade);
-    			if (!empty($grade)) {
-    				$teacherIds[] = $teacher->id;
-    			}
-    			$teachers = $teachers->whereIn('id', $teacherIds)->get();
-    		}
-    	}
+        $request = $request->all();
+        $teachers = Teacher::orderBy('id','ASC');
+        if (isset($request['grades'])) {
+            $teacherIds = [];
+            foreach ($teachers as $key => $teacher) {
+                $grade = array_intersect($request['grades'], $teacher->grade);
+                if (!empty($grade)) {
+                    $teacherIds[] = $teacher->id;
+                }
+                $teachers = $teachers->whereIn('id', $teacherIds)->get();
+            }
+        }
 
-    	if (isset($request['subjects'])) {
-    		$teacherIds = TeacherSubjects::whereIn('subject_id', $request['subjects'])->pluck('teacher_id');
-    		$teachers =  $teachers->whereIn('id', $teacherIds);
-    	}
-    	$grades = Grades::get();
-    	$subjects = Subject::all();
-    	$teachers = $teachers->paginate(10);
+        if (isset($request['subjects'])) {
+            $teacherIds = TeacherSubjects::whereIn('subject_id', $request['subjects'])->pluck('teacher_id');
+            $teachers =  $teachers->whereIn('id', $teacherIds);
+        }
+        $grades = Grades::get();
+        $subjects = Subject::all();
+        $teachers = $teachers->paginate(10);
 
-    	return view('home', compact('grades','subjects','teachers','request'));
+        return view('home', compact('grades','subjects','teachers','request'));
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/HomeController.php</br>**Line**: 38</br>No space found after comma in function call